test/release/examples/primers/forallLoops.chpl

```diff
@@ -320,31 +326,59 @@ Task-Private Variables
 ----------------------
 
 A task-private variable is similar to an in-intent or ref-intent
-shadow variable in that it is initialized at the beginning of its
-task and deallocated at the end of the task. However, a task-private
-variable is initialized without regard to any outer variable.
+shadow variable in that it is set up at the beginning of its
+task before the task executes any loop iterations, and deallocated
+at the end of the task after the task completes all "its" iterations.
```

Comment:
This somehow feels a bit more wordy. Also, I know we only have TPVs with loops right now, but should we be mentioning loops explicitly here?

Maybe: "...in that their lifetimes roughly correspond to the lifetime of the corresponding loop". Additionally, each task created by the parallel iterator gets its  own set of task-private variables. Unlike shadow variables, TPVs do not correspond to outer variables."